src/relay/backend/te_compiler.cc

```diff
@@ -593,6 +593,14 @@ Pass LowerTensorExpr(TargetMap targets, DeviceMap device_context_map,
   return CreateFunctionPass(pass_func, 0, "LowerTensorExpr", {});
 }
 
+/*!
+ * \brief Obtain the Target from the device type.
+ * If homogenous compilation, this will return the only target.
+ * If heteregenous compilation, this will select associated using the targets_ Map.
```

Comment:
```suggestion
 * If heteregenous compilation, this will select the associated target using the targets_ Map.
```

Reply:
I am making this change in the header file